src/DataModel/IData.ts

```diff
@@ -0,0 +1,6 @@
+import {IMetadata} from "./IMetadata";
+
+export interface IData extends Object
```

Comment:
Renamed with `IWebResource`. I wanted to leave `IResource` to match the term used in the Hydra spec.